README.md

```diff
@@ -93,6 +93,12 @@ Change the environment variables in the [provided example](./docker-compose.yml)
 
 Note that in order to avoid writing credentials directly into the file, you can create a `db_secrets.env` and use the `env_file` directive (see the [docs](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option)).
 
+### Docker secrets
+
+As an alternative to passing sensitive information via environment variables, _FILE may be appended to some of the environment variables, causing the initialization script to load the values for those variables from files present in the container. In particular, this can be used to load passwords from Docker secrets stored in /run/secrets/<secret_name> files.
```

Comment:
As there are only 5 of them, I think it's better to specify them

Reply:
added a list supported _FILE variables